src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+
+interface UserInfoProps {
+  user: { id: number; name: string; email: string };
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
Ensure that the `user.email` is a valid email address to prevent issues with the mailto link. Consider adding validation or default values if there's a possibility of missing or malformed email data.